cmd/kubelet/app/server.go

```diff
@@ -355,6 +255,24 @@ func makeEventRecorder(s *componentconfig.KubeletConfiguration, kubeDeps *kubele
 	}
 }
 
+// Returns the most likely identifer for the current Node
+func curNodeIdentifier(s *options.KubeletServer, kd *kubelet.KubeletDeps) (string, error) {
+	hostname := nodeutil.GetHostname(s.HostnameOverride)
+	if kd != nil && kd.Cloud != nil {
+		instances, ok := kd.Cloud.Instances()
+		if !ok {
+			err := fmt.Errorf("failed to get instances from cloud provider, can't determine nodename")
+			return "", err
```

Comment:
I ended up removing the whole function, turned out to be a duplicate of `getNodeName`